test/cluster-store.js

```diff
@@ -2,7 +2,7 @@ var cluster = require('cluster');
 var expect = require('chai').expect;
 var ioServer = require('socket.io');
 var ioClient = require('socket.io-client');
-var debug = require('debug')('test');
+var debug = require('debug')('strong-cluster-socket.io-store:test');
```

Comment:
Ah, I did not realise `DEBUG=*:test` works too.

> The scheme above is consistent with the rest of the strong-* repos

I see, in that case it's probably worth doing it your way. Feel free to rebase 4d4e964 away.